tests/testthat/tests-is_climatic_data.R

```diff
@@ -0,0 +1,43 @@
+library(testthat)
+library(R6)
```

Comment:
It looks like it doesn't like you running `library(R6)`.

Can you remove this from here?

Then below where you say `R6Class("MockData", ...)`, we can change that to `R6::R6Class("MockData", ...)`?